lib/avatar.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class Avatar extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return const CircleAvatar(
+      backgroundImage: AssetImage('assets/images/jose.ramos.jpeg'),
```

Comment:
For you to take into consideration, Avatar could receive the image path as parameter, in case you want to reuse that in the future